modules/api/app/utils/to_string.go

```diff
@@ -63,9 +63,9 @@ func ArrStringsToString(arr []string) (result string, err error) {
 	result = ""
 	for indx, a := range arr {
 		if indx == 0 {
-			result = fmt.Sprintf("\"%v\"", a)
+			result = fmt.Sprintf("'%v'", a)
```

Comment:
i think `"` or `'` are both workable in mysql select statment such as `select * from table where xx in ()`  @tsingliu